checkstyle/checkstyle-suppressions.xml

```diff
@@ -25,7 +25,10 @@
   <suppress checks="ConstantName"
             files="ApplicationStatus.java"
             lines="26-29"/>
+  <!-- don't require FQDNs to be used in API javadocs -->
```

Comment:
I don't understand this comment and what is it relevant to?

Reply:
I will clarify this comment, similar to above.